Content/UsePrivacyMode.md

```diff
@@ -0,0 +1,17 @@
+# Privacy Mode
+
+_UsePrivacyMode_ is a box type preset available since v1.0.5 / 5.55.5.
```

Comment:
```suggestion
_UsePrivacyMode_ is a box type preset available since v1.0.0 / 5.55.0.
```

Source: [CHANGELOG.md#100--5550---2021-11-17](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#100--5550---2021-11-17)